tests/framework/installer/ceph_manifests_v1.1.go

```diff
@@ -1670,32 +1668,12 @@ spec:
 
 // GetCleanupPod gets a cleanup Pod manifest
 func (m *CephManifestsV1_1) GetCleanupPod(node, removalDir string) string {
```

Comment:
The integration test is hung now, looks like I was wrong about removing this. For now let's restore this in the 1.1 test manifests and separately I'll look into the upgrade test bug that caused this failure.